assembl/scripts/pshell.py

```diff
@@ -0,0 +1,44 @@
+import sys
+
+from pyramid.util import DottedNameResolver
+from pyramid.settings import aslist
+from pyramid.scripts.pshell import PShellCommand
+
+
+def main(argv=sys.argv, quiet=False):
+    command = PShellCommandA(argv, quiet)
+    try:
+        return command.run()
+    except:
+        import pdb; pdb.post_mortem()
+
+
+class PShellCommandA(PShellCommand):
```

Comment:
A bit in https://github.com/assembl/assembl/pull/106/commits/675360d3c0c5f0f8a5fcbe2ca8b2dafd0e4d8a3c